lib/matplotlib/figure.py

```diff
@@ -1192,12 +1196,12 @@ def subplots_adjust(self, left=None, bottom=None, right=None, top=None,
             The height of the padding between subplots,
             as a fraction of the average Axes height.
         """
-        if self.get_constrained_layout():
-            self.set_constrained_layout(False)
+        if (self._layout_engine is not None and
```

Comment:
do you want to be consistent between `self._layout_engine` and `self.get_layout_engine()`?

Reply:
Good idea.  Changed bare instances to the setter/getter.